packages/space-opera/src/components/camera_settings/camera_settings.ts

```diff
@@ -246,14 +241,6 @@ export class CameraSettings extends ConnectedLitElement {
     return html`
     <me-expandable-tab tabName="Camera Setup" .open=${true}>
       <div slot="content">
-        <me-checkbox id="cam-controls-checkbox" label="Interactive camera"
```

Comment:
So we're just removing the ability to change this entirely? (Making the assumption that 99% of the time, people want this enabled.)

Reply:
Yeah, we already only expose a subset of our API as UX, and this didn't feel deserving of being included. Likewise, I have AR on by default instead of selectable.